### PR TITLE
Sgt back end

### PR DESCRIPTION
**Pull request by rskidmore1**
image

### CODE REVIEW

File: sgt-back-end/index.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  select *
+  from "grades"`;
+
+  db.query(sql)
+    .then(results => {
+      const grade = results.rows;
+      if (!grade) {
+        res.status(200).json({
+          noResults: 'There are no grades'
+        });
```
Comment (thebearingedge) [inline sgt-back-end/index.js:24]: In JS, arrays are always truthy (even if empty) so this `if` will never run.

File: sgt-back-end/index.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  const vals = [req.body.name, req.body.course, req.body.score];
+  if (req.body.name === undefined || req.body.course === undefined || req.body.score > 100) {
+    res.status(400).json({
+      error: 'One or more inputs is mal-formed'
+    });
+  } else {
+    db.query(sql, vals)
+      .then(results => {
+        const grade = results.rows;
+        res.status(201).json(grade);
```
Comment (thebearingedge) [inline sgt-back-end/index.js:63]: The client should receive a JSON object instead of an array.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great just a couple minor changes.

**Review by thebearingedge — APPROVED** [review]

**Comment by rskidmore1** [comment]
muchthanks

**Pull request merged**